core/src/main/scala/org/apache/spark/SparkStatusTracker.scala

```diff
@@ -104,4 +106,34 @@ class SparkStatusTracker private[spark] (sc: SparkContext) {
       }
     }
   }
+
+  /**
+   * Returns a list of all known executors, represented by string with format: "host:port"
+   */
+  def getExecutors(): Array[String] = {
```

Comment:
we can expose both, but I'd rather call it something more explicit like `getExecutorHostPort` or something. Elsewhere in Spark I would think `getExecutors: Array[String]` returns the executor IDs.